### ISSUE TITLE
New URL Format

### ISSUE THREAD

**Issue by smaragdus** [coder-chenzhi/EasyTabURL-ChromeExtension#4]
Hello,

I find **Easy Tab URL** extension very handy and useful, good work, thank you.

Yet I would like to suggest a tiny enhancement- you may add Options for the extension and add new format for generating and copying URLs. Now **Easy Tab URL** uses this format:

`Title	URL`

I would prefer this format:

`Title`
`URL`

For example, in stead of the current:

single line format (Title and URL in the same line):

- Wikipedia, the free encyclopedia	https://en.wikipedia.org/wiki/Main_Page

There is an option to use:

two lines format (Title and URL in separate lines):

- Wikipedia, the free encyclopedia
https://en.wikipedia.org/wiki/Main_Page

I suppose that this options will not be very hard to be implemented.

**Comment by coder-chenzhi** [comment]
@smaragdus Thanks for your advice. What you mentioned can be achieved by common text editors, like notepad, notepad++, Sublime Text, etc. You just need to replace all "http" by "\nhttp".

**Comment by smaragdus** [comment]
@coder-chenzhi 

You are right but I would prefer automatic solution allowing me to copy and paste directly without any editing. If one day you decide to add options to **Easy Tab URL** you may add an option for 'Title and URL' format - 'single line' - 'Title & URL in one line' or 'separate lines' - 'Title & URL in separate lines'.

**Comment by smaragdus** [comment]
@coder-chenzhi 

I forgot to mention one more thing- 'Load' works but you may add also 'Paste' command which gets text from clipboard and opens URLs in new tabs- just like 'Load' but without the need to use 'Open' dialog and select a text file. I think that 'Paste' command might be faster than 'Load' command.

One more thing about 'Load' command, when I select a text file, **Easy Tab URL** opens the saved links right but it also opens an empty tab:

`chrome-extension://fccbmnpilngffifjlblcnaaeoahbamfp/`

with the following error message:

```
Your file was not found
It may have been moved or deleted.
ERR_FILE_NOT_FOUND
```

Screen:

![snipaste-2019-06-11_004355](https://user-images.githubusercontent.com/9025757/59229277-a0505600-8be2-11e9-9618-ff1c3010b674.png)

Do you get this error when using 'Load' command?
